.travis.yml

```diff
@@ -1,19 +1,107 @@
+sudo: required
+dist: trusty
 language: go
-sudo: false
 go:
   - "1.10.3"
+
 # import forks to the correct location
 go_import_path: github.com/VertebrateResequencing/wr
+
+services:
+  - docker
+
+cache:
+  directories:
+    - $GOPATH/pkg/
+
+env:
+  global:
+    - KUBERNETES_VERSION=v1.10.0
+    # Move Kubernetes config files to the appropriate place and adjust permissions.
+    - CHANGE_MINIKUBE_NONE_USER=true
+    - DEP_VERSION="0.5.0"
+
 before_install:
-  - go get github.com/modocache/gover
-  - go get github.com/mattn/goveralls
```

Comment:
Still need gover and goveralls for my tests

Reply:
Ah, yes. 
I'm not sure how to invoke that for my tests, especially to try and get coverage across packages
(Lots of the methods in `pod.go` I'm testing in the scheduler/deployment controller tests (Port forwarding, attach cmd etc)